client/src/app/core/core-services/stable.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+import { BehaviorSubject, Observable } from 'rxjs';
+
+import { Deferred } from '../promises/deferred';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class StableService {
+    public get isStable(): Promise<void> {
+        return this.stable;
+    }
+
+    public get booted(): Observable<boolean> {
```

Comment:
This is only needed in the interceptor - move the BehaviorSubject there

Reply:
Also: booted is another terminology that indicates, that OpenSlides has started up - do not use this wording twice.